src/game/actions.ts

```diff
@@ -12,7 +12,7 @@ export interface MoveAction {
 
 export interface MagicAction {
   type: "MAGIC";
-  spell: Spell;
+  spell: "Spell";
```

Comment:
Shouldn't this be a type so we can load various types of Spells?

Reply:
This was actually fixed here: https://github.com/TheUnblockedDemo/2048/commit/9e73f2f3a4a9fed154e17c6eb4cd462f46406ebd

Was missing an import for this enum that already exists.